perfplot/main.py

```diff
@@ -110,7 +158,7 @@ def bench(
         # round up to nearest integer
         resolution = -int(-resolution // 1)  # typically around 10 (ns)
 
-    timings = numpy.empty((len(kernels), len(n_range)), dtype=numpy.uint64)
+    timings = numpy.empty((len(kernels), len(n_range)), dtype=numpy.float64)
```

Comment:
It'd be great it the timings could be stored natively (in `uint64`) and only at plot time be converted to the appropriate unit.

Reply:
Resolved by: 21588a07bcaac2828a40a5faacb47f450f063a5a